slider/__main__.py

```diff
@@ -24,10 +24,20 @@ def main():
     help='Show a progress bar?',
     default=True,
 )
-def library(beatmaps, recurse, progress):
+@click.option(
+    '--skip-exceptions/--no-skip-exceptions',
+    help='Skip beatmap files that cause exceptions rather than exiting?',
+    default=True,
```

Comment:
I would make this default to `False` personally. This might hide errors a user would otherwise catch. The error message we throw could include a mention to this option to point users in the right direction should they want to enable this.